yodeploy/test_integration/samples/uwsgi-app/deploy/hooks.py

```diff
@@ -0,0 +1,20 @@
+import getpass
+
+import mock
+
+from os.path import join
+
+from yodeploy.hooks.uwsgi import UwsgiHostedApp
+from yodeploy.test_integration.helpers import tests_dir
+
+
+class Hooks(UwsgiHostedApp):
+    # these overrides are only needed for testing
```

Comment:
Probably needless comment here.

Reply:
Personal pref is to always bake comments into readable code:
```diff
- class Hooks(UwsgiHostedApp):
+ class HooksWithTestSetup(UwsgiHostedApp):
```

Reply:
> Personal pref is to always bake comments into readable code:
> ```diff
> - class Hooks(UwsgiHostedApp):
> + class HooksWithTestSetup(UwsgiHostedApp):
>```

That indeed looks better!